python/lesson3/tutorial.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Discovering Flask via Python and pip
+---
+
+An alternative to *Ruby's Sinatra* is the lesser known *Flask* (run using Python).
+Both are web application frameworks. Similar to the MVC (model-view-controller)
+pattern but for dynamic and smaller applications and sites.
+
+Knowing both can be advantageous in work scenarios, understanding
+different programming and computer science conventions or thoughts.
+
+# Why Flask and Sinatra?
+Both are small and encourage building applications from ground up using
+dependencies and components.
```

Comment:
Do think 'small' could point out the file-size or something a bit more appropriate. Hmm. Worth more thinking around.